File: SCHEDULER/src/main/webapp/WEB-INF/web/static/js/scheduler/schedulerutil.js
var schedulerutil = {};

schedulerutil.isBlank = function(input) {
	return (undefined === input || null === input || '' == input.trim());
}

schedulerutil.isNotBlank = function(input) {
	return (undefined !== input && null !== input && '' !== input.trim());
}

schedulerutil.isNullOrUndefined = function(input) {
	return (undefined === input || null === input);
}

File: SCHEDULER/src/main/java/com/erstegroup/lio/utils/SchedulerUtil.java
package com.erstegroup.lio.utils;

import org.apache.commons.lang3.StringUtils;

public class SchedulerUtil {

	private SchedulerUtil() {
		
	}
	
	public static boolean isBlank(final CharSequence cs) {
		return StringUtils.isBlank(cs);
	}
	
	public static boolean isNotBlank(final CharSequence cs) {
		return StringUtils.isNotBlank(cs);
	}
	
	public static String trim(final String str) {
		return StringUtils.trim(str);
	}
	
	@SafeVarargs
	public static <T> String join(final T ...elements) {
		return StringUtils.join(elements);
	}
	
}
